src/utils/taxonomy.ts

```diff
@@ -1,20 +1,55 @@
+import { Request } from "express";
+import { CONTACT_US_THEMES, PATH_NAMES } from "../app.constants";
+
 export enum TaxonomyLevel1 {
   BLANK = "",
   AUTHENTICATION = "authentication",
 }
 
 export enum TaxonomyLevel2 {
   BLANK = "",
+  FEEDBACK = "feedback",
+  GUIDANCE = "guidance",
 }
 
 export type Taxonomy = {
   taxonomyLevel1: TaxonomyLevel1;
   taxonomyLevel2: TaxonomyLevel2;
 };
 
-export function getRequestTaxonomy(): Taxonomy {
+export function getRequestTaxonomy(req: Request): Taxonomy {
   return {
     taxonomyLevel1: TaxonomyLevel1.AUTHENTICATION,
-    taxonomyLevel2: TaxonomyLevel2.BLANK,
+    taxonomyLevel2: getTaxonomyLevel2(req),
   };
 }
+
+function getTaxonomyLevel2(req: Request) {
+  let taxonomyLevel2 = TaxonomyLevel2.BLANK;
+
+  if (isFeedbackTaxonomy(req)) {
+    taxonomyLevel2 = TaxonomyLevel2.FEEDBACK;
+    if (isGuidanceTaxonomy(req)) {
+      taxonomyLevel2 = TaxonomyLevel2.GUIDANCE;
+    }
+  }
+
+  return taxonomyLevel2;
+}
+
+const isFeedbackTaxonomy = (req: Request): boolean => {
+  return [
+    PATH_NAMES.CONTACT_US,
+    PATH_NAMES.CONTACT_US_FROM_TRIAGE_PAGE,
+    PATH_NAMES.CONTACT_US_FURTHER_INFORMATION,
+    PATH_NAMES.CONTACT_US_QUESTIONS,
+    PATH_NAMES.CONTACT_US_SUBMIT_SUCCESS,
```

Comment:
I'm wondering if there's any way to make this less flimsy (would be easy to miss adding this change in when we add a new contact page).

Wondering if e.g. adding a differentiation at the level of path names (grouping the contact form names into a separate object away from other path names, so that there'd be more indication when adding a new path name about where it should go), so here we'd be able to say "if in any contact us path names...". Not sure if that makes sense outside of my head!

Doesn't necessarily need fixing (if we do want to fix) in this PR, can be a follow up to reduce review complexity.

Reply:
Cool, I'm happy to have it in a follow up :-)

Reply:
HI folks 👋🏽, thanks for seeking my input on this. I completely agree with all you've both said. One thing I do wonder about with the potential approach above where `taxonomyLevel2` is a property of the path is if there's going to be circumstances where there might not be a 1:1 relationship between a path and taxonomy values (for example in the Strategic App). I'm not entirely sure what the analytics needs will be going forward, but I'd anticipate there will be an ask for us to disambiguate between the two. This leads me to think the approach of calculating the taxonomy might give us more flexibility.